notebooks/datasets/nldas_visualize_rawdata.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>What does this mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>